src/Entities/Boat.h

```diff
@@ -31,8 +31,27 @@ class cBoat :
 	virtual void HandleSpeedFromAttachee(float a_Forward, float a_Sideways) override;
 
 	cBoat(double a_X, double a_Y, double a_Z);
-} ;
 
+	int GetTimeSinceLastHit(void) const { return m_TimeSinceLastHit; }
```

Comment:
It's used to control how the boat wobbles visually.  It's set to 10 when the boat is hit (attacked), and decremented by 1 each tick if it's greater than 0.

Since it's used as a countdown clientside, decreasing it each tick probably would be easier to handle than using the time it was hit.  Even still, `TimeSinceLastHit` really doesn't describe how it is used clientside - it's more how much the boat is wobbling (MCP does use `TIME_SINCE_HIT` to describe it, which is probably why wiki.vg calls it that).  

Minecarts have the exact same type of code for their "Shaking power" value (set to 10 on hit, decremented each tick).